third_party/xkcp/BUILD.xkcp.bazel

```diff
@@ -0,0 +1,64 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "nist_fips_202",
+    srcs = [
+        "lib/high/Keccak/FIPS202/KeccakHash.c",
+        "lib/high/Keccak/FIPS202/SimpleFIPS202.c",
+        "lib/high/Keccak/KeccakSponge.c",
+        "lib/high/Keccak/KeccakSponge.inc",
+        "lib/low/KeccakP-1600/ref-64bits/KeccakP-1600-reference.c",
+    ],
+    hdrs = [
+        "config.h",
+        "lib/common/align.h",
+        "lib/common/brg_endian.h",
+        "lib/high/Keccak/FIPS202/KeccakHash.h",
+        "lib/high/Keccak/FIPS202/SimpleFIPS202.h",
+        "lib/high/Keccak/KeccakSponge.h",
+        "lib/high/Keccak/KeccakSponge.inc",
+        "lib/low/KeccakP-1600/ref-64bits/KeccakP-1600-SnP.h",
+        "lib/low/KeccakP-1600/ref-64bits/KeccakP-1600-reference.h",
```

Comment:
I think you may want to move some of the `.h`/`.inc` files listed here to the `srcs` list. See the [official Bazel documentation](https://bazel.build/reference/be/c-cpp#hdrs) on what to list where. To summarize, you want to place `#include`d files in the `srcs` list that are only included by header/source files that are specific to this library. If the header file is designed to be included by other targets that include this library as a dep (i.e., headers that make up the public API for this library), then you should put it in the `hdrs` list, otherwise it should go in `srcs`. I haven't looked into the source code of the lib you are vendoring in, but I'm guessing the `.inc` are examples of things that should go in `srcs`.

Reply:
same applies to this lib below

Reply:
Thanks, makes sense. Fixing this.

Reply:
The fixes LGTM, thanks @ballifatih.